api/pre_process_data.py

```diff
@@ -176,3 +176,62 @@ def populationAndArableLand(country="World"):
                 "Arable Land": units["Arable Land"],
             },
         }
+
+    def _preprocess_data(file1, file2, indicator1, indicator2, info, country="World"):
```

Comment:
As per our discussion, this utility function was supposed to be a private function of the class (denoted with `__`, two underscores in the beginning).
This means, that this function can be called only inside the `PreprocessData` class. Namely in the specific preprocess functions, such as:

```
def populationAndArableLand(self, country="World"):
        return self.__preprocess_data(
            file1="population",
            file2="arable_land",
            indicator1="Population",
            indicator2="Arable Land",
            info="population_and_arable_land",
        )
```

Reply:
This change also means, that now the arguments of the `__preprocess_data` method also have to change:

```
 def __preprocess_data(
        self, file1, file2, indicator1, indicator2, info, country="World"
    ):
```